test_run.sh

```diff
@@ -186,6 +186,43 @@ test_junit_test_errors_when_no_tests_found() {
   exit 0
 }
 
+test_resources() {
+  RESOURCE_NAME="resource.txt"
+  OUTPUT_JAR="bazel-bin/test/src/main/scala/scala/test/resources/$TARGET.jar"
+  FULL_TARGET="test/src/main/scala/scala/test/resources/$TARGET.jar"
+  bazel build $FULL_TARGET
+  jar tf $OUTPUT_JAR | grep $RESOURCE_NAME
+  RET=$?
+  if [ $RET -ne 0 ]; then
+    echo "Missing ${RESOURCE_NAME} in jar based on $JAR_DESCRIPTION"
+    exit 1
+  fi
+}
+
+scala_library_jar_with_srcs_includes_direct_file_resources(){
```

Comment:
You are right, we don't it's already tested with `test:RunMoreScalaLibBinary`. Removing it.